scripts/run_ida_script.py

```diff
@@ -16,6 +21,17 @@
 logger = logging.getLogger(__name__)
 
 
+def script_path_to_args(script_path):
```

Comment:
can we use shlex for this?

https://docs.python.org/2.7/library/shlex.html#shlex.split

Reply:
can, it's nice